pypopquiz/io.py

```diff
@@ -24,13 +25,14 @@ def verify_input(input_data: Dict) -> None:
         "properties": {
             "round": {"type": "number"},
             "theme": {"type": "string"},
+            "spacers": {"type": "boolean"},
             "questions": {
                 "type": "array",
                 "minItems": 1,
                 "additionalProperties": False,
                 "items": {
                     "type": "object",
-                    "required": ["artist", "title", "sources", "question", "answer"],
+                    "required": ["artist", "title", "sources", "question"],
```

Comment:
Why is the `answer` no longer mandatory to have? The code still assumes it is given, right?

Reply:
I switched it off during testing, and forgot to switch it back on. I'll re-add it.